src/docker/Dockerfile

```diff
@@ -30,6 +30,8 @@ RUN apt-get update && apt-get --assume-yes install software-properties-common \
       && apt-get --assume-yes install libssl-dev \
       && apt-get --assume-yes install gcc-aarch64-linux-gnu \
       && apt-get --assume-yes install g++-aarch64-linux-gnu \
+      && apt-get --assume-yes install gcc-riscv64-linux-gnu \
+      && apt-get --assume-yes install g++-riscv64-linux-gnu \
```

Comment:
This Dockerfile is referenced by [`src/docker/docker-compose.yaml`](https://github.com/apache/commons-crypto/pull/264/files#diff-a24c82f935acd3ecf6920e0147bd0f8d171c10687789ed202ef960063c52c2fb), and CI uses that `src/docker/docker-compose.yaml` to run [`src/docker/build.sh`](https://github.com/apache/commons-crypto/pull/264/files#diff-810d630fb6f62bdcc13127f70e147a23857dbeaf1daa6552173279939e4bd57d), which itself is going to call `mvn package` which triggers the cross-compilation for all targets. `gcc-riscv64-linux-gnu` and `g++-riscv64-linux-gnu` is thus required to be in this Dockerfile, or it will be missing when building the package on CI.

Reply:
That's just to simplify the build, and the time gained would be minimal anyway.